HW04/Alexey_Zalyotov/spec/my_module_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'my_module'
+
+class Array
+  include MyModule
+end
+
+describe '#my_each' do
+  subject { Array.new([1, 2, 3]) }
```

Comment:
Normally, we should use method from your test description `describe '#my_each'` in our subject 
```suggestion
  subject { Array.new([1, 2, 3]).my_each(&block) }
```
And would be better to have the opportunity to update block from the specific test cases.

Reply:
Done